src/modules/zerion-api/hooks/useAssetChart.ts

```diff
@@ -0,0 +1,20 @@
+import { useQuery } from '@tanstack/react-query';
+import { ZerionAPI } from 'src/modules/zerion-api/zerion-api.client';
+import { type Params } from '../requests/asset-get-chart';
+
+export function useAssetChart(
+  params: Params,
+  {
+    suspense = false,
```

Comment:
Is `true` used anywhere?

Reply:
And why not? :)

Reply:
I think, I just copied the api for the hook from the other similar hooks. 
Can be simplified

Reply:
I don't think that hooks like this should blocks other content from rendering, so suspense is always false